control/preimage/src/treasury_commands.rs

```diff
@@ -0,0 +1,245 @@
+use crate::{constants::*, Context, TreasuryProposal2024Args};
+
+use crate::helpers::utility_force_batch;
+use crate::relay_runtime::api::runtime_types::{
+    pallet_treasury,
+    polkadot_runtime_common::impls::VersionedLocatableAsset,
+    staging_xcm::v4::{
+        asset::AssetId, junction::Junction, junctions::Junctions, location::Location,
+    },
+    xcm::VersionedLocation,
+};
+use crate::relay_runtime::RuntimeCall as RelayRuntimeCall;
+use polkadot_runtime_constants::currency::UNITS;
+use polkadot_runtime_constants::time::DAYS;
+
+pub const USDC_UNITS: u8 = 1000000;
+
+#[derive(Copy, Clone)]
+enum TreasuryAsset {
+    DOT,
+    USDC,
+}
+
+impl Into<Location> for TreasuryAsset {
+    fn into(self) -> Location {
+        match self {
+            TreasuryAsset::DOT => Location {
+                parents: 1,
+                interior: Junctions::Here,
+            },
+            TreasuryAsset::USDC => Location {
+                parents: 0,
+                interior: Junctions::X2([
+                    Junction::PalletInstance(50),
+                    Junction::GeneralIndex(1337),
+                ]),
+            },
+        }
+    }
+}
+
+struct Spend {
+    name: &'static str,
+    asset: TreasuryAsset,
+    amount: u128,
+    delay: Option<u32>,
+}
+
+pub const fn dot(amount: u128) -> u128 {
+    amount * UNITS
+}
+
+pub const fn usdc(amount: u128) -> u128 {
+    amount * 1_000_000
+}
+
+pub const SPENDS: [Spend; 23] = [
+    Spend {
+        name: "Operational & Development costs",
```

Comment:
Operational & Auditing costs

Reply:
is just a comment basically, does not go onchain.